SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+Think you've found a vulnerability in the Uptane Standard? Please review the following details for responsible disclosure.
+
+## Supported Versions
+
+We release updates to the Uptane specification to address security vulnerabilities. Which versions are eligible to receive updates are listed below:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.2.0   | :white_check_mark: |
+| 1.1.0   | :white_check_mark: |
+| 1.0.1   | :white_check_mark: |
+| 1.0.0   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+Please report (suspected) security vulnerabilities to `uptane-security [AT] googlegroups [DOT] com`. You will receive an acknowledgement from us within 72 hours. Keep in mind this project is a community based project of volunteers and response times could vary. If the issue is confirmed, we will provide a response within 90 days.
```

Comment:
Who is behind that email address? This can be discussed elsewhere if we don't want that to be publicly disclosed right now. It's just the first I've heard of that and I'm curious who is taking that responsibility on.

Reply:
The question of where these reports go is a good one. I don't mind monitoring and referring these reports, but ideally whoever gets those responses should be able to address the problem. I think its worth bringing this up in one of our meetings. I'm listed as a maintainer for the projects on the website, but it would be good to have a technical person listed as well to address security flaws.